google/resource_container_node_pool_test.go

```diff
@@ -513,6 +536,58 @@ resource "google_container_node_pool" "np" {
 }`, cluster, np)
 }
 
+func testAccContainerNodePool_maxPodsPerNode(cluster, np string) string {
+	return fmt.Sprintf(`
+resource "google_compute_network" "container_network" {
+	name = "container-net-%s"
+	auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "container_subnetwork" {
+	name                     = "${google_compute_network.container_network.name}"
+	network                  = "${google_compute_network.container_network.name}"
+	ip_cidr_range            = "10.0.36.0/24"
+	region                   = "us-central1"
+	private_ip_google_access = true
+
+	secondary_ip_range {
+		range_name    = "pod"
+		ip_cidr_range = "10.0.0.0/19"
+	}
+
+	secondary_ip_range {
+		range_name    = "svc"
+		ip_cidr_range = "10.0.32.0/22"
+	}
+}
+
+resource "google_container_cluster" "cluster" {
+	name = "%s"
+	zone = "us-central1-a"
+	initial_node_count = 3
+
+	network = "${google_compute_network.container_network.name}"
+	subnetwork = "${google_compute_subnetwork.container_subnetwork.name}"
+	private_cluster = true
```

Comment:
I'm going to go ahead and just assume that all of these attributes are actually needed in order to set the field (including `cidr_blocks = []` which makes me sad)

Reply:
It's true.  :(  GKE's interdependencies are very complex.